Classes/Core/Controllers/FLEXTableViewController.m

```diff
@@ -324,16 +324,10 @@ - (void)setupToolbarItems {
         self.middleLeftToolbarItem,
         UIBarButtonItem.flex_flexibleSpace,
         self.middleToolbarItem,
-        UIBarButtonItem.flex_flexibleSpace,
-        self.bookmarksToolbarItem,
-        UIBarButtonItem.flex_flexibleSpace,
-        self.openTabsToolbarItem,
     ];
     
     for (UIBarButtonItem *item in self.toolbarItems) {
-        [item _setWidth:60];
-        // This does not work for anything but fixed spaces for some reason
-        // item.width = 60;
```

Comment:
I love this comment on the source 😅

Reply:
> Out of curiosity, what is the rationale for the NLCache name?

Nick Lockwood, the author of the OSCache.